frontend/src/components/AvailabilityViewer/AvailabilityViewer.tsx

```diff
@@ -101,7 +101,12 @@ const AvailabilityViewer = ({ times, people, table }: AvailabilityViewerProps) =
                 people: peopleHere,
               })
             }}
+            onClick={() => {
+              const clipboardMessage = `${t('group.clipboard_message', { date: cell.label })}:\n${peopleHere.join(',\n')}`
```

Comment:
Is there a reason why you want a new line after each name? I think it might look cleaner like this:

```
People available on June 6, 2023 at 11:00 AM:
Midas, mike, Test
```

Reply:
I just feel it might be better to have the paragraph if you want to place it in CSV, but i guess it could be a niche situation